CRM/Upgrade/Incremental/php/FourSeven.php

```diff
@@ -404,6 +404,18 @@ public function upgrade_4_7_23($rev) {
     $this->addTask(ts('Upgrade DB to %1: SQL', array(1 => $rev)), 'runSql', $rev);
   }
 
+  /**
+   * Upgrade function.
+   *
+   * @param string $rev
+   */
+  public function upgrade_4_7_25($rev) {
+    $this->addTask('CRM-20927 - Add Generic Column for other data', 'addColumn',
```

Comment:
@seamuslee001 Thanks, this is a good idea/safer code-convention.

Nitpick: from the perspective of an admin watching the upgrade UI, this would display a message "CRM-20927 - Add Generic Column for other data", which seems meaningless at a glance. The capitalization also looks pretty arbitrary.

Maybe something like "CRM-20927 - Add column to 'civicrm_menu' for additional metadata"?